frontend/.storybook/mockData.ts

```diff
@@ -0,0 +1,85 @@
+import { auditLogFactory } from "../src/fixtures/auditLog";
+import { destructionListAssigneesFactory } from "../src/fixtures/destructionList";
+import { FIXTURE_SELECTIELIJSTKLASSE_CHOICES } from "../src/fixtures/selectieLijstKlasseChoices";
+import { userFactory, usersFactory } from "../src/fixtures/user";
+import { zaaktypeChoicesFactory } from "../src/fixtures/zaaktypeChoices";
+
+export const MOCKS = {
+  AUDIT_LOG: {
+    url: "http://localhost:8000/api/v1/destruction-lists/00000000-0000-0000-0000-000000000000/auditlog/",
+    method: "GET",
+    status: 200,
+    response: auditLogFactory(),
+  },
+  DESTRUCTION_LIST_MAKE_FINAL: {
+    url: "http://localhost:8000/api/v1/destruction-lists/00000000-0000-0000-0000-000000000000/make_final",
+    method: "POST",
+    status: 201,
+    response: undefined,
+  },
```

Comment:
I think this should be `response: {}`

![Screenshot from 2024-11-15 12-44-21](https://github.com/user-attachments/assets/099e53ac-9623-4c52-8323-7e8fb1637502)